whois-client/pom.xml

```diff
@@ -16,6 +16,10 @@
             <groupId>net.ripe.db</groupId>
             <artifactId>whois-rpsl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.ripe.db</groupId>
+            <artifactId>whois-commons</artifactId>
+        </dependency>
```

Comment:
Can we move the client to whois-api rather in adding this dependency?

Reply:
Yes! `whois-client` should only be used for connecting clients to Whois, not to other APIs !

Reply:
true